com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Procedural/Noise/VoronoiNode.cs

```diff
@@ -0,0 +1,162 @@
+using System.Collections.Generic;
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class VoronoiNode : IStandardNode
+    {
+        static string Name = "Voronoi";
+        static int Version = 1;
+        public static NodeDescriptor NodeDescriptor => new(
+            Version,
+            Name,
+            "VoronoiDeterministic",
+            functions: new FunctionDescriptor[] {
+                new(
+                    "Unity_Voronoi_RandomVector_Deterministic",
```

Comment:
`Shader error in 'Test': undeclared identifier 'Unity_Voronoi_RandomVector_Deterministic_float' at line 476 (on d3d11)` same as above

Reply:
This should now be fixed.